### PR TITLE
NEM integration

### PR DESCRIPTION
**Pull request by cassade**

### CODE REVIEW

File: src/Lykke.Job.Nem/Services/NemJob.cs [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+                    }
+
+                    var transfer = tx as TransferTransaction ?? 
+                        throw new InvalidOperationException($"Wrongly parsed transaction {tx.TransactionInfo.Hash}");
+
+                    _log.Info($"New transfer detected", tx.TransactionInfo);
+
+                    var blockNumber = (long)transfer.TransactionInfo.Height;
+                    var blockTime = _nemesis.AddSeconds(transfer.TransactionInfo.TimeStamp);
+                    var memo = (transfer.Message as PlainMessage)?.GetStringPayload();
```
Comment (KonstantinRyazantsev) [inline src/Lykke.Job.Nem/Services/NemJob.cs]: Add TrimStart+TrimEnd. Users often forgot to remove whitespace symbols after copy/paste

File: src/Lykke.Job.Nem/Startup.cs [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+        public IServiceProvider ConfigureServices(IServiceCollection services)
+        {
+            return services.BuildServiceProvider<AppSettings>(options =>
+            {
+                options.SwaggerOptions = _swaggerOptions;
+
+                options.Logs = logs =>
+                {
+                    logs.AzureTableName = "NemJobLog";
+                    logs.AzureTableConnectionStringResolver = settings => settings.NemJob.Db.LogsConnString;
```
Comment (KonstantinRyazantsev) [inline src/Lykke.Job.Nem/Startup.cs:35]: Add custom Slack channels bcn and bcn-imprtnt. disable spam guarding by channelOptions.SpamGuard.DisableGuarding(); and enable health notifications by channelOptions.IncludeHealthNotifications();

File: src/Lykke.Job.Nem/Services/NemJob.cs (hunk 1 above)
Comment (cassade) [reply src/Lykke.Job.Nem/Services/NemJob.cs]: Fixed

File: src/Lykke.Job.Nem/Startup.cs (hunk 2 above)
Comment (cassade) [reply src/Lykke.Job.Nem/Startup.cs:35]: Fixed

File: src/Lykke.Job.Nem/Lykke.Job.Nem.csproj [hunk 3]
```diff
@@ -24,7 +24,8 @@
     <PackageReference Include="Lykke.Sdk" Version="5.7.0" />
     <PackageReference Include="Microsoft.AspNetCore" Version="2.2.0" />
     <PackageReference Include="Microsoft.AspNetCore.StaticFiles" Version="2.2.0" />
-    <PackageReference Include="Lykke.Service.BlockchainApi.Sdk" Version="1.2.0-beta.1" />
+    <PackageReference Include="Lykke.Service.BlockchainApi.Sdk" Version="1.2.0-beta.3" />
```
Comment (KonstantinRyazantsev) [inline src/Lykke.Job.Nem/Lykke.Job.Nem.csproj]: Should be not a beta version

Comment (cassade) [reply src/Lykke.Job.Nem/Lykke.Job.Nem.csproj]: Sure. I was going to change versions after testing in DEV, before merging to test.

**Review by KonstantinRyazantsev — APPROVED** [review]

**Pull request merged**